rs/core/corpora_cli/src/commands/init.rs

```diff
@@ -1,4 +1,22 @@
-/// Run the `init` command
-pub fn run() {
-    println!("init command executed with no arguments.");
+use crate::context::Context;
+
+pub fn run(ctx: &Context) {
+    println!("Initializing a new corpus...");
+
+    let corpus_name = "example_corpus";
```

Comment:
TODO: get corpus_name from config or git and build the tarball dynamically from ls-files with type text.